ce/ce/amf/installer.ts

```diff
@@ -220,6 +220,14 @@ class GitCloneNode extends Installer implements GitInstaller {
     this.setMember('subdirectory', value);
   }
 
+  get espidf() {
+    return this.flags.has('espidf');
+  }
+
+  set espidf(value: boolean) {
+    this.flags.set('espidf', value);
```

Comment:
Doesn't this need to unset if false and set if true? The "get" says true on "has".

Reply:
set's implementation does that 

> in flags.ts
```  typescript
set(flag: string, value: boolean) {
    this.assert(true);
    if (value) {
      this.node.add(new Scalar(flag));
    } else {
      this.node.delete(flag);
    }
  }
```

Reply:
Sounds good! I thought it was just an array rather than a thing...